src/RestServer/Controllers/WalletController.cs

```diff
@@ -0,0 +1,258 @@
+            var wallet = _walletSessions[sessionId];
+            var accounts = new List<WalletAddressModel>();
+            foreach (var account in wallet.GetAccounts())
+                accounts.Add(new WalletAddressModel()
+                {
+                    Address = account.Address,
+                    HasKey = account.HasKey,
+                    Label = account.Label,
+                    WatchOnly = account.WatchOnly,
+                });
+            return Ok(accounts.ToArray());
+        }
+
+        [HttpPost("{session}/send")]
+        public IActionResult WalletSendTo(
+            [FromRoute(Name = "session")]
+            string session,
+            [FromBody]
+            WalletSendModel model)
+        {
+            if (Guid.TryParse(session, out Guid sessionId) == false) return BadRequest();
+            if (_walletSessions.ContainsKey(sessionId) == false) return NotFound();
+            if (ModelState.IsValid == false) return BadRequest();
+            if (UInt160.TryParse(model.To, out var scriptHash) == false)
+                scriptHash = model.To.ToScriptHash(_neosystem.Settings.AddressVersion);
+            var wallet = _walletSessions[sessionId];
+            using var snapshot = _neosystem.GetSnapshot();
+            var descriptor = new AssetDescriptor(snapshot, _neosystem.Settings, model.AssetId);
+            var amount = new BigDecimal(model.Amount, descriptor.Decimals);
+            if (amount.Sign <= 0) return BadRequest(nameof(model.Amount));
+            var tx = wallet.MakeTransaction(snapshot, new[]
+            {
+                new TransferOutput()
+                {
+                    AssetId = model.AssetId,
+                    Value = amount,
+                    ScriptHash = scriptHash,
+                }
+            });
+            if (tx == null) return BadRequest("Tranasction");
+            var transContext = new ContractParametersContext(snapshot, tx, _neosystem.Settings.Network);
+            wallet.Sign(transContext);
+            if (transContext.Completed == false)
+                return Content(transContext.ToJson().ToString(), MediaTypeNames.Application.Json);
+            tx.Witnesses = transContext.GetWitnesses();
+            // What if this is completed? Process and sign again?
+            if (tx.Size > 1024)
+            {
+                var calFee = tx.Size * NativeContract.Policy.GetFeePerByte(snapshot) + unchecked((long)NativeContract.GAS.Factor);
+                if (tx.NetworkFee < calFee) tx.NetworkFee = calFee;
+
+            }
+            if (tx.NetworkFee > _settings.MaxTransactionFee) return BadRequest("The necessary fee is more than the MaxTransactionFee, this transaction has failed. Please increase your MaxTransactionFee value.");
+
+            // Why do this again? What if its completed already? Copied from RPC Server...
+            transContext = new ContractParametersContext(snapshot, tx, _neosystem.Settings.Network);
```

Comment:
Any reason why we need to do this again? In addition, what if previous transaction was completed? Why resign it? Note this is copied from RPC Server.